akamai/cache-client_test.go

```diff
@@ -237,33 +250,35 @@ func TestBigBatchPurge(t *testing.T) {
 		"secret",
 		"accessToken",
 		"production",
+		2,
 		3,
 		time.Second,
+		time.Millisecond*32,
 		log,
 		metrics.NoopRegisterer,
 	)
 	test.AssertNotError(t, err, "Failed to create CachePurgeClient")
 
-	var urls []string
+	var urls [][]string
 	for i := 0; i < 250; i++ {
-		urls = append(urls, fmt.Sprintf("http://test.com/%d", i))
+		urls = append(urls, []string{fmt.Sprintf("http://test.com/%d", i)})
 	}
 
 	stoppedAt, err := client.Purge(urls)
 	test.AssertNotError(t, err, "Purge failed with 201 response")
 	test.AssertEquals(t, stoppedAt, 250)
 
 	// Add a malformed URL.
-	urls = append(urls, "http:/test.com")
+	urls = append(urls, []string{"http:/test.com"})
 
 	// Add 10 more valid entries.
 	for i := 0; i < 10; i++ {
-		urls = append(urls, fmt.Sprintf("http://test.com/%d", i))
+		urls = append(urls, []string{fmt.Sprintf("http://test.com/%d", i)})
 	}
 
 	stoppedAt, err = client.Purge(urls)
 	test.AssertError(t, err, "Purge succeeded with a malformed URL")
-	test.AssertEquals(t, stoppedAt, 200)
+	test.AssertEquals(t, stoppedAt, 250)
```

Comment:
This value changes because instead of purging in batches of `100` we're purging in batches of `2`. This last batch contains just 11 URLs, the first is the malformed URL and the latter `10` were added on line `275`. I've added some comments and additional assertions to prove this out.